actionmailbox/MIT-LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Basecamp, LLC
+Copyright (c) 2020 Basecamp, LLC
```

Comment:
Probably this should be 2018-2020

```suggestion
Copyright (c) 2018-2020 Basecamp, LLC
```

Reply:
Fixed it @kamipo . Marking comment as resolved. Thanks